RtspServer/RtspServer.cs

```diff
@@ -1474,6 +1474,7 @@ internal virtual void MaintainServer(object state = null)
                     }), Common.Extensions.Thread.ThreadExtensions.MinimumStackSize)
                     {
                         Priority =  m_ServerThread.Priority,
+                        Name = "Maintenance",
```

Comment:
Lets include the ID along with the name e.g. Id + Maintenance or "Maintenance" + Id